### PR TITLE
[api-minor] Moving text layer rendering into src/display and better "text" testing.

### PR DESCRIPTION
**Pull request by yurydelendik**
Combines page rendering and text layer in the "text" tests.

(thuluthfont-text test was removed due to invalid unicode characters in the text nodes)

### CODE REVIEW

File: test/text_layer_test.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  -webkit-transform-origin: 0% 0%;
+  -moz-transform-origin: 0% 0%;
+  -o-transform-origin: 0% 0%;
+  -ms-transform-origin: 0% 0%;
+  transform-origin: 0% 0%;
+  border: solid 1px rgba(255, 0, 0, 0.5);
+  background-color: rgba(255, 255, 32, 0.1);
+  -webkit-box-sizing:border-box;
+  -moz-box-sizing:border-box;
+  box-sizing:border-box;
```
Comment (Rob--W) [inline test/text_layer_test.css]: Add space after `:`, 3x.

File: test/test_manifest.json [hunk 2]
```diff
@@ -326,12 +326,6 @@
       "lastPage": 1,
       "about": "The same file as issue2337."
     },
-    {  "id": "thuluthfont-text",
-       "file": "pdfs/ThuluthFeatures.pdf",
-       "md5": "b7e18bf7a3d6a9c82aefa12d721072fc",
-       "rounds": 1,
-       "type": "text"
-    },
```
Comment (Rob--W) [inline test/test_manifest.json]: Shouldn't the file itself (and the entry in `.gitignore`) be removed as well?

Comment (yurydelendik) [inline test/test_manifest.json]: we have "eq" test for it.

**Comment by yurydelendik** [comment]
Made #textLayer=visible match to new "text" test output.

/botio-linux preview

**Comment by timvandermeij** [comment]
I would say that this fixes #4576.

**Pull request merged**